source/main/assets/export.tex

```diff
@@ -166,6 +166,15 @@
 \providecommand{\tightlist}{%
   \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
 
+$if(csl-refs)$
```

Comment:
We need a short comment explaining why this csl-refs-clause is here, just so we don't wonder in the future!

Reply:
Half the stuff in the Pandoc template is pretty opaque if you don't know the internals of what Pandoc / LaTeX. I suggest just tracking the upstream changes to the data template as closely as possible.

Reply:
Absolutely, but we know for sure that this environment was introduced in 2.8, so I just want this comment here so that it's easier to spot.

Reading a short comment is always simpler than having to dig up the full version history of Pandoc's template.

Reply:
I added a comment. Is this what you had in mind?

Reply:
Yes, thank you very much! :)